h2o-core/src/main/java/water/DKV.java

```diff
@@ -219,4 +224,31 @@ static private Value get( Key key, boolean blocking ) {
     RPC<TaskGetKey> tgk = TaskGetKey.start(home,key);
     return blocking ? TaskGetKey.get(tgk) : null;
   }
+
+  public static final class RetainKeysTask extends MRTask<RetainKeysTask> {
```

Comment:
retain? is that a good name - this is actually deleting objects :)

Reply:
Not sure. It retains some and deletes others. Let's call it `ClearDKVTask` ?

Reply:
Renamed.